src/chess_widget.rs

```diff
@@ -193,27 +198,32 @@ impl Widget<String> for ChessWidget {
                     }
                 }
             }
+            Event::KeyDown(key_event) => match key_event.key {
+                Key::ArrowLeft => {
+                    println!("left arrow pressed down")
+                }
+                Key::ArrowRight => {
+                    println!("right arrow pressed down")
+                }
+                Key::Character(ref c) => {
+                    println!("'{}' pressed down", c.to_uppercase())
+                }
+                _ => {}
+            },
             _ => {}
         }
     }
 
-    fn lifecycle(
-        &mut self,
-        _ctx: &mut LifeCycleCtx,
-        _event: &LifeCycle,
-        _data: &String,
-        _env: &Env,
-    ) {
-    }
+    fn lifecycle(&mut self, ctx: &mut LifeCycleCtx, event: &LifeCycle, data: &String, env: &Env) {}
```

Comment:
So the underscores get rid of the unused variable warning right?

Reply:
yeah since we have a bunch of other warnings right now, it's good to have to remind us to either mark as unused, or implement it. I'm not sure what exactly we will or wont use so